torch/_dynamo/variables/tensor.py

```diff
@@ -690,6 +689,8 @@ def redistribute_fn_with_prim_types(x):
                 ), "Unreachable - See unimplemented for lambdas above"
             tx.output.side_effects.register_hook(self, fn_var, handle_variable)
             return handle_variable
+        elif name == "requires_grad_":
+            unimplemented("Tensor.requires_grad_")
```

Comment:
Should we avoid graph-breaking on the case when this is a no-op? If tensor already requires grad and the input is True, or vice versa

Reply:
+1